src/client/components/GameCardComponent/ButtonComponent/ButtonComponent.stories.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ButtonComponent from './ButtonComponent';
+
+export default { title: 'Button Component' };
+
+export const Component = () => <ButtonComponent buttonTitle="Trial button" />;
```

Comment:
Maybe seems a bit redundant to name the prop `buttonTitle` as the prop is in used in the context of the Button component already? could just be named title? 🙌  but up to you if you want to change it or not